jobs/build/publish-rpms/Jenkinsfile

```
@@ -0,0 +1,47 @@
+#!/usr/bin/env groovy
+
+node {
+    checkout scm
+    buildlib = load( "pipeline-scripts/buildlib.groovy" )
+    commonlib = buildlib.commonlib
+    commonlib.describeJob("publish-rpms", """
+        <h2>Automatically publishing rpms to mirror</h2>
+    """)
+
+    properties(
+        [
+            disableResume(),
+            buildDiscarder(
+                logRotator(
+                    artifactDaysToKeepStr: '',
+                    artifactNumToKeepStr: '',
+                    daysToKeepStr: '',
+                    numToKeepStr: ''
+                )
+            ),
+            [
+                $class : 'ParametersDefinitionProperty',
+                parameterDefinitions: [
+                    string(
+                        name: 'BUILD_VERSION',
+                        description: 'Target release version',
+                        trim: true,
+                    ),
+                ]
+            ],
+            disableResume(),
+            disableConcurrentBuilds()
+        ]
+    )
+    commonlib.checkMock()
+
+    version = params.BUILD_VERSION
+    scripts_dir = "${env.WORKSPACE}/hacks/rhel-7-deps-mirroring/collect-deps.sh"
+    mirror_dir = "/srv/pub/openshift-v4/dependencies/rpms/${version}-beta"
+￼   withEnv(["PATH+SCRIPTS=${scripts_dir}"]) {
+￼       sh "collect-deps.sh ${version}"
+￼       sh "rm -rf /srv/pub/openshift-v4/dependencies/rpms/${version}-beta/*"
+        sh "scp -r ${version}-beta/ use-mirror-upload:$mirror_dir"
+        sh "ssh use-mirror-upload 'createrepo --database $mirror_dir;/usr/local/bin/push.pub.sh openshift-v4/dependencies/rpms/${version}-beta -v'"
```

use commonlib.shell and one giant string. also take out the unprintable unicode...
```suggestion
        commonlib.shell """
            collect-deps.sh ${version}
            rm -rf /srv/pub/openshift-v4/dependencies/rpms/${version}-beta/*
            scp -r ${version}-beta/ use-mirror-upload:${mirror_dir}
            ssh use-mirror-upload 'createrepo --database ${mirror_dir};/usr/local/bin/push.pub.sh openshift-v4/dependencies/rpms/${version}-beta -v'
        """
```